### PR TITLE
feat: make python2/3 compatible

### LINKED ISSUE #3: Python 3+ compliance
Make the library Python 3+ compliant

### PR DESCRIPTION
**Pull request by jrconlin**
Closes #3

### CODE REVIEW

**Comment by safwanrahman** [comment]
So the difference between ``byte`` and ``unicode`` is the issue?

**Comment by jrconlin** [comment]
@safwanrahman Pretty much. Python3 is a lot more strict about string types.

**Comment by safwanrahman** [comment]
With small python 3 knowledge, I will check it tomorrow.

**Comment by jrconlin** [comment]
cool, I'm having @bbangert look it over as well, just to make sure I didn't screw up something

File: pywebpush/__init__.py [hunk 1]
```diff
@@ -86,6 +86,11 @@ def __init__(self, subscription_info):
             the client.
 
         """
+        # Python 2 v. 3 hack
```
Comment (bbangert) [inline pywebpush/__init__.py]: I'd recommend using the six library rather than custom hacks:
https://pythonhosted.org/six/#binary-and-text-data

**Pull request merged**